src/app/signin/page.tsx

```diff
@@ -26,34 +30,20 @@ export default function SignIn() {
   };
 
   return (
-    <main className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#2e026d] to-[#15162c] text-white">
+    <main className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-primary-300 to-secondary-200 text-white">
       <div className="container flex flex-col items-center justify-center gap-12 px-4 py-16 ">
-        <h1 className="font-bold">Sign in</h1>
+        <div className="h1-bold">Sign in</div>
```

Comment:
use h1, it's semantic thing.
btw `h1-bold` look really weird, doesn't h1 should be always bold anyway?

Reply:
in global.css there are h1 and h1-bold